src/ez/DB.java

```diff
@@ -163,6 +180,10 @@ public static boolean isValidName(String name) {
   }
 
   public DB transaction(Runnable r) {
+    return transaction(IsolationLevel.SERIALIZABLE, r);
+  }
+
+  public DB transaction(IsolationLevel level, Runnable r) {
```

Comment:
optional parameter should go 2nd, not 1st.

Reply:
I had done it like this because lambdas can be long so front loading all params before the lambda was more readable. But nonetheless I still implemented this suggestion.